Python/ds/Double_ended_queue.py

```diff
@@ -0,0 +1,117 @@
+# Flexible Dequeue
```

Comment:
Add a beginner pitch about what Flexible Dequeue is. This will help others know about the DS/Algo being implemented.

Reply:
Okay1

Reply:
Done!